### PR TITLE
Creates contributing.md for ember times

### PR DESCRIPTION
**Pull request by Mi6u3l**
## What it does
- Creates contributing.md for ember times

### CODE REVIEW

**Comment by tylerturdenpants** [comment]
I think this is great. It formalizes the process and is easy to look up. Hopefully @jessica-jordan has some input

File: source/blog/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- Fork the repo
+- Clone the repo in your own machine
+- Commit changes to your own branch
+- Push the changes
+- Create a pull request to the new blog post branch on the original repo
+
+To update your fork with the new weekly branch you can do:
+
+```sh
+git fetch upstream
```
Comment (jessica-jordan) [inline source/blog/CONTRIBUTING.md]: This is super useful :+1: Might it make sense to add a note on how to set `upstream` and `origin` on your local machine as well? I think not everyone might have the `upstream` pointed to the original repo and the `origin` pointed to their fork of the website repo

**Comment by kennethlarsen** [comment]
This looks really great! If you want to add a more detailed description of the workflow in Goodbits then come by my desk @Mi6u3l and I'll demo it to you.

File: source/blog/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+- The championing editor of the week creates a branch with the MD file containing next week blog post
+- Editors will then pull request against that branch with changes to that file
+- Once the new blog post file is complete, the newly created branch is merged back into the website
+- As a last step, the content is copied over to [Goodbits](https://the-emberjs-times.ongoodbits.com/), which is the tool currently being used to send the newsletter and manage its subscriptions
+
+The Reader's Questions section of the newsletter is usually answered by core team members, but anyone can give a helping hand. 
+The answers are posted on [discuss.emberjs.com](https://discuss.emberjs.com/) and linked back to the weekly blog post.
+
+### Goodbits
```
Comment (jessica-jordan) [inline source/blog/CONTRIBUTING.md]: Super, it's really useful to have all of this outlined in a nut shell :+1:

**Pull request merged**